components/binding/cmd/manager/main.go

```diff
@@ -49,7 +55,11 @@ func main() {
 		logger.SetLevel(log.WarnLevel)
 	}
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	config := ctrl.GetConfigOrDie()
+	cli, err := client.New(config, client.Options{Scheme: scheme})
```

Comment:
do not create new client, manager gives you one:
```go
mgr.GetClient()
```
see e.g. method `controller.SetupTargetKindReconciler`

Reply:
Unfortunately, you can't use the manager's client before it started